chapters/stack/functions/drills/tasks/string_print/support/print_string.asm

```diff
@@ -0,0 +1,21 @@
+%include "../utils/printf32.asm"
+
+section .data
+    mystring db "This is my string", 0
+
+section .text
+
+extern puts
+extern printf
+global main
+main:
+    push ebp
+    mov ebp, esp
+
+    PRINTF32 `[PRINTF32]: %s\n[PUTS]: \x0`, mystring
+
+
+    ; TODO: call puts on string
+
+    leave
+    ret
```

Comment:
I strongly recommand a new line at the end of file. See [1]
[1] https://gist.github.com/OleksiyRudenko/d51388345ea55767b7672307fe35adf3